test/nyc-integration.js

```diff
@@ -49,7 +49,7 @@ t.test('--check-coverage fails when check-coverage command is used rather than f
   return testSuccess(t, {
     args: [process.execPath, './half-covered.js']
   }).then(() => testFailure(t, {
-    args: ['check-coverage', '--lines', '51', process.execPath, './half-covered.js']
+    args: ['check-coverage', '--lines', '51']
```

Comment:
Could we rename this test to something like `check-coverage command is equivalent to the flag`